src/components/wallet-selector/wallet-selector.ts

```diff
@@ -41,4 +47,8 @@ export class WalletSelectorComponent extends ActionSheetParent {
       this.dismiss(optionClicked);
     }
   }
+
+  public readMore() {
+    this.logger.debug('Read more clicked'); // TODO: add a link or remove this read more
```

Comment:
update link or hide button to read more option for now...

Reply:
Done